build.sc

```diff
@@ -1,35 +1,32 @@
-
 import mill._
 import mill.scalalib._
 import mill.scalalib.publish._
 import mill.scalajslib._
 import mill.scalanativelib._
 import mill.modules._
 import mill.scalanativelib.api.{LTO, ReleaseMode}
+import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version_mill0.9:0.1.1`
+import de.tobiasroeser.mill.vcs.version.VcsVersion
+
+val dottyCustomVersion = sys.props.get("dottyVersion")
 
-val scala212  = "2.12.13"
-val scala213  = "2.13.4"
-val scala3    = "3.0.0-RC1"
-val scalaJS06 = "0.6.33"
-val scalaJS1  = "1.4.0"
-val scalaNative = "0.4.0"
+val scala213 = "2.13.4"
+val scala212 = "2.12.13"
 
-val dottyCustomVersion = Option(sys.props("dottyVersion"))
+val scalaVersions = scala213 :: scala212 :: "3.0.0-RC1" :: dottyCustomVersion.toList
+val scala2Versions = scalaVersions.filter(_.startsWith("2."))
 
-val scala2JVMVersions = Seq(scala212, scala213)
-val scalaJVMVersions = scala2JVMVersions ++ Seq(scala3) ++ dottyCustomVersion
+val scalaJS1 = "1.4.0"
 
-val scalaJSVersions = Seq(
-  (scala212, scalaJS06),
-  (scala213, scalaJS06),
-  (scala212, scalaJS1),
-  (scala213, scalaJS1)
-)
+val scalaJSVersions = for {
+  scalaV <- scala2Versions
+  scalaJSV <- Seq("0.6.33", scalaJS1)
+} yield (scalaV, scalaJSV)
 
-val scalaNativeVersions = Seq(
-  (scala212, scalaNative),
-  (scala213, scalaNative)
-)
+val scalaNativeVersions = for {
+  scalaV <- scala2Versions
+  scalaNativeV <- Seq("0.4.0")
+} yield (scalaV, scalaNativeV)
```

Comment:
Why change these? IMO the explicit version is easier to understand and follow

Reply:
Did it for consistency with the other repos. Reverted since this PR has another purpose either way.